app/config/events.yml

```diff
@@ -0,0 +1,47 @@
+        - Surfnet\Stepup\Configuration\Event\IdentityProvidersUpdatedEvent
+        - Surfnet\Stepup\Configuration\Event\InstitutionConfigurationRemovedEvent
+        - Surfnet\Stepup\Configuration\Event\NewConfigurationCreatedEvent
+        - Surfnet\Stepup\Configuration\Event\NewInstitutionConfigurationCreatedEvent
+        - Surfnet\Stepup\Configuration\Event\RaLocationAddedEvent
+        - Surfnet\Stepup\Configuration\Event\RaLocationContactInformationChangedEvent
+        - Surfnet\Stepup\Configuration\Event\RaLocationRelocatedEvent
+        - Surfnet\Stepup\Configuration\Event\RaLocationRemovedEvent
+        - Surfnet\Stepup\Configuration\Event\RaLocationRenamedEvent
+        - Surfnet\Stepup\Configuration\Event\ServiceProvidersUpdatedEvent
+        - Surfnet\Stepup\Configuration\Event\ShowRaaContactInformationOptionChangedEvent
+        - Surfnet\Stepup\Configuration\Event\SraaUpdatedEvent
+        - Surfnet\Stepup\Configuration\Event\UseRaLocationsOptionChangedEvent
+        - Surfnet\Stepup\Identity\Event\AppointedAsRaaEvent
+        - Surfnet\Stepup\Identity\Event\AppointedAsRaEvent
+        - Surfnet\Stepup\Identity\Event\CompliedWithUnverifiedSecondFactorRevocationEvent
+        - Surfnet\Stepup\Identity\Event\IdentityAccreditedAsRaaEvent
+        - Surfnet\Stepup\Identity\Event\IdentityEmailChangedEvent
+        - Surfnet\Stepup\Identity\Event\GssfPossessionProvenEvent
+        - Surfnet\Stepup\Identity\Event\IdentityAccreditedAsRaEvent
+        - Surfnet\Stepup\Identity\Event\IdentityCreatedEvent
+        - Surfnet\Stepup\Identity\Event\EmailVerifiedEvent
+        - Surfnet\Stepup\Identity\Event\CompliedWithVettedSecondFactorRevocationEvent
+        - Surfnet\Stepup\Identity\Event\CompliedWithVerifiedSecondFactorRevocationEvent
+        - Surfnet\Stepup\Identity\Event\IdentityForgottenEvent
+        - Surfnet\Stepup\Identity\Event\IdentityRenamedEvent
+        - Surfnet\Stepup\Identity\Event\LocalePreferenceExpressedEvent
+        - Surfnet\Stepup\Identity\Event\InstitutionsAddedToWhitelistEvent
+        - Surfnet\Stepup\Identity\Event\PhonePossessionProvenEvent
+        - Surfnet\Stepup\Identity\Event\RegistrationAuthorityRetractedEvent
+        - Surfnet\Stepup\Identity\Event\RegistrationAuthorityInformationAmendedEvent
+        - Surfnet\Stepup\Identity\Event\InstitutionsRemovedFromWhitelistEvent
+        - Surfnet\Stepup\Identity\Event\U2fDevicePossessionProvenEvent
+        - Surfnet\Stepup\Identity\Event\SecondFactorVettedEvent
+        - Surfnet\Stepup\Identity\Event\VerifiedSecondFactorRevokedEvent
+        - Surfnet\Stepup\Identity\Event\WhitelistCreatedEvent
+        - Surfnet\Stepup\Identity\Event\UnverifiedSecondFactorRevokedEvent
+        - Surfnet\Stepup\Identity\Event\VettedSecondFactorRevokedEvent
+        - Surfnet\Stepup\Identity\Event\WhitelistReplacedEvent
+        - Surfnet\Stepup\Identity\Event\YubikeySecondFactorBootstrappedEvent
+        - Surfnet\Stepup\`Identity\Event\YubikeyPossessionProvenEvent
```

Comment:
The typo was fixed in the other PR (I was messing around with some hotkeys), but the `class_exists` check is indeed a good idea to detect errors early.